beacon_node/src/cli.rs

```diff
@@ -955,6 +955,16 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .takes_value(true)
                 .default_value("180")
         )
+        .arg(
+            Arg::with_name("checkpoint-sync-state-id")
```

Comment:
would you mind adding a CLI test for this 😄 🙏

Reply:
mark pointed out this doesn't end up in config so doesnt work with our CLI tests